image/templates/helm/shared/templates/02-scanner-05-network-policy.yaml

```diff
@@ -1,6 +1,6 @@
 {{- include "srox.init" . -}}
 
-{{- if not ._rox.scanner.disable -}}
+{{- if and ._rox.system.enableNetworkPolicies (not ._rox.scanner.disable) -}}
```

Comment:
nit: `._rox.system.enableNetworkPolicies` could be on their own line for better readability

Reply:
Done.